cate/webapi/start.py

```diff
@@ -98,28 +98,33 @@ def service_factory(application):
 # }
 
 def create_application(user_root_path: str = None):
-    application = Application([
-        ('/_static/(.*)', StaticFileHandler, {'path': FigureManagerWebAgg.get_static_file_path()}),
-        ('/mpl.js', MplJavaScriptHandler),
-
-        (url_pattern('/mpl/download/{{base_dir}}/{{figure_id}}/{{format_name}}'), MplDownloadHandler),
-        (url_pattern('/mpl/figures/{{base_dir}}/{{figure_id}}'), MplWebSocketHandler),
+    default_url_root = "/"
+    # replace default url_root with /user/username/ if running in Cate Hub context.
+    url_root = os.environ.get("JUPYTERHUB_SERVICE_PREFIX", default_url_root)
```

Comment:
Is "JUPYTERHUB_SERVICE_PREFIX" a well-known Juypter environment variable? Or is it your invention?

Reply:
If it is a well-known Juypter environment variable, then this is fine.
If not, I suggest to call that variable "CATE_WEBAPI_PREFIX".

Reply:
JUPYTERHUB_SERVICE_PREFIX is Jupyterhub passes as environment variable to each spawned notebook container. we can assign it internally to CATE_WEBAPI_PREFIX but pointless i think.